src/test/java/org/example/integration/JobRolesIntegrationTest.java

```diff
@@ -79,4 +79,116 @@ void getJobRoleById_shouldReturnAJobRole() {
 
     }
 
+    @Test
+    void get10JobRoles_shouldReturn10JobRoles() {
+
+        Client client = APP.client();
+
+
+        LoginRequest loginRequest = new LoginRequest(
+                "admin@kainos.com",
+                "wlSNgEn5dCBM59jnbeH+txKWn36Vt6QScELcAa5ZBNduqSY16JAl2hqeGsZrmpG0kdb9+ILMoCJVB3er8ZoCJI9o26IM83UfnJtTT3p7cRgOUxsU0iMHgkI9KdQpDim6"
+
+        );
+        String token = "Bearer " + client
+                .target("http://localhost:8080/api/auth/login")
+                .request()
+                .post(Entity.json(loginRequest))
+                .readEntity(String.class);
+
+        Response response = client
+                .target("http://localhost:8080/api/job-roles?page=1&limit=10")
```

Comment:
Using a hardcoded variable on the address will break the integration tests as live uses https and local uses http.

Reply:
Fixed